manifests/setup/cows.pp

```diff
@@ -116,7 +114,7 @@
   # Earlier debian and ubuntu versions won't have the scripts for newers versions for
   # debootstrap.  We also lay down the script for cumuluslinux which is currently a
   # copy of wheezy.
-  debbuilder::util::file_on_disk { ['jessie', 'precise', 'quantal', 'raring', 'saucy', 'trusty', 'utopic', 'wheezy', 'CumulusLinux-2.2']:
+  debbuilder::util::file_on_disk { ['trusty', 'utopic', 'vivid', 'CumulusLinux-2.2']:
```

Comment:
Hmm, I guess we still need to do this for platforms that aren't available in the debootstrap package on older platforms as well, in case community members are using older debian/ubuntu platforms to build on